test/pummel/test-watch-file.js

```diff
@@ -44,10 +47,17 @@ function watchFile() {
 
 watchFile();
 
+function changeFile() {
+  const fd = fs.openSync(f, 'w+');
+  fs.writeSync(fd, 'xyz\n');
+  fs.closeSync(fd);
+}
 
-const fd = fs.openSync(f, 'w+');
-fs.writeSync(fd, 'xyz\n');
-fs.closeSync(fd);
+changeFile();
+const interval = setInterval(changeFile, 5000);
```

Comment:
It wouldn’t hurt to set this to a lower value, e.g. `100`, right? I don’t quite see why this test has to be in pummel, and if we set it to a lower value, I feel like it should run quite quickly and could also just be moved to `parallel`

Reply:
I agree the test does not need to be in pummel. This can be set more often, but I believe the default for `watchFile()` is to check for changes every 5-ish seconds. Maybe `1000` would be good so we're recreating the file about 5 times before exiting rather than 50?

Reply:
> I agree the test does not need to be in pummel.

Although let's land this and make sure it resolves the flakiness across platforms before moving it out?

Reply:
Sure, that works for me :+1: